projects/aca-content/src/lib/components/files/files.component.ts

```diff
@@ -71,6 +78,14 @@ export class FilesComponent extends PageComponent implements OnInit, OnDestroy {
   ngOnInit() {
     super.ngOnInit();
 
+    this.extensions
```

Comment:
can be moved to base class, also, why removing certain actions in code? should be removed from the extensions json file?